src/main/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/configs/TestConfig.groovy

```diff
@@ -39,10 +35,17 @@ class TestConfig extends AbstractDescribableImpl<TestConfig> implements Expandab
     }
 
     TestConfig(TestConfig config) {
+        this.forceConfigurationReload = config.forceConfigurationReload
+        //setting paths and constants to empty in case forceConfigurationReload is set to true
+        if(this.forceConfigurationReload) {
```

Comment:
* had to remove `static`from the configOption getter, otherwise tests like `config round trip` would have thrown: `java.lang.NoSuchMethodException: Property 'configOption' has no getter method in class 'class de.tracetronic.jenkins.plugins.ecutestexecution.configs.TestConfig'`
* had to set `forceConfigurtionReload` to false in if branch, otherwise a combination in the snippetGenerator of `[forceConfigurationReload: true, tbcPath: 'test.tbc', tcfPath: 'test.tcf']` would have been possible